addStudent.php

```diff
@@ -33,8 +34,8 @@
 
 
             #Adding student records based on the information the user added into the "add" fields
-            $stmt = $db->prepare("INSERT INTO Student (First_Name, Last_Name, Gender, Birth_Date, Address, City, State, Zip, Ethnicity, School, Program, Permission_Slip, Birth_Certificate, Reduced_Lunch_Eligible, IEP) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
-            $stmt->bind_param('sssssssisssiiii', $fname, $lname, $gender, $dob, $address, $city, $state, $zip, $ethnicity, $school, $program, $permission_slip, $birth_certificate, $reduced_lunch_eligible, $iep);
+            $stmt = $db->prepare("INSERT INTO student (First_Name , Middle_Name, Last_Name, Suffix,Gender, Birth_Date, Address, City, State, Zip, Ethnicity, School, Permission_Slip, Birth_Certificate, Reduced_Lunch_Eligible, IEP) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,?)");
+            $stmt->bind_param('sssssssisssiiii', $fname , $mname, $lname, $suffix, $gender, $dob, $address, $city, $state, $zip, $ethnicity, $school, $permission_slip, $birth_certificate, $reduced_lunch_eligible, $iep);
```

Comment:
i think the problem is with the bind params, we have 16 variables being passed in, and there are only 15 bind params listed here.  Also the 8th bind param for city is currently an "i" for integer and I believe its suppose to be an "s" for string

Reply:
Maybe it was I was half asleep last night and maybe this morning but I'm counting 16 variables for the parameters/bind params but now I realize my error. I'll be fixing it after work today.